Brewfile

```diff
@@ -0,0 +1,5 @@
+cask "AdoptOpenJDK/openjdk/adoptopenjdk8"
+
+tap "guardian/devtools"
```

Comment:
I think it auto taps when you use a cask or brew that is fully qualified.

Reply:
Oh really?! That makes things so much easier!